### ISSUE TITLE
Virtual auto-getprops set in constructor on overriden type crash Bridge

### ISSUE THREAD

**Issue by michaelcheers** [bridgedotnet/Bridge#2792]
### Steps To Reproduce

https://deck.net/969279bee4684479a957d804022a43d8
https://dotnetfiddle.net/lSgZ3D

```c#
public class Program
{
    public static void Main()
    {
        Console.WriteLine(new B().Name);
    }
}

public class A
{
    public virtual string Name{get;}
    
    public A ()
    {
        Name = "Fail";
    }
}

public class B : A
{
    public override string Name => "Pass";
}
```

### Expected Result
Console outputs: **Pass**

### Actual Result
```
TypeError: Cannot set property Name of # which has only a getter
    at Class.ctor (https://deck.net/RunHandler.ashx?h=41910029:14:27)
    at new Class (https://deck.net/resources/js/bridge/bridge.js?16.0.0-beta2:2543:46)
    at Function.Main (https://deck.net/RunHandler.ashx?h=41910029:21:13)
    at https://deck.net/resources/js/bridge/bridge.js?16.0.0-beta2:3144:39
```

**Comment by geoffreymcgill** [comment]
Thanks for the report and excellent sample. We will fix this right away.

**Comment by geoffreymcgill** [comment]
Here's another similar test we were using:

https://deck.net/bce3746f396685b7378ea9b6c19fdd01

```csharp
public class A
{
    public virtual string Name { get; }

    public A()
    {
        Name = "Fail";
        Console.WriteLine(this.Name);
    }
}

public class B : A
{
    public override string Name { get; }
}

public class App
{
    public static void Main()
    {
        var b = new B();
    }
}
```

This defect has been fixed and will be included in the upcoming **16.0.0-beta3**.
